homeassistant/components/input_number.py

```diff
@@ -19,18 +19,25 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-DOMAIN = 'input_slider'
+DOMAIN = 'input_number'
 ENTITY_ID_FORMAT = DOMAIN + '.{}'
 
 CONF_INITIAL = 'initial'
 CONF_MIN = 'min'
 CONF_MAX = 'max'
+CONF_MODE = 'mode'
 CONF_STEP = 'step'
 
+MODE_BOTH = 'both'
```

Comment:
We should not support both, that seems like an odd use case?

Reply:
@balloob it certainly looks wonky with both I'll grant you that. I could see some people still liking the option to set an exact value and use a slider though - but I can remove the support for both if that is preferred.

Reply:
Let's remove it. It's always easier to add things than to remove them 👍